panda/src/downloader/config_downloader.h

```diff
@@ -24,6 +24,18 @@
 #include "configVariableFilename.h"
 #include "configVariableList.h"
 
+/**
+ * For some reason in msys those two macros are not defined correctly in the
+ * header file ws2tcpip.h
+ */
+#ifndef AI_ADDRCONFIG
+#define AI_ADDRCONFIG 0x00000400
```

Comment:
Might not want to duplicate this ourselves, and especially not in the .h (should do it in a .cxx so we don't affect downstream projects); I see from the MinGW32-w64 headers that this is wrapped in a `(_WIN32_WINNT >= 0x0600)`. We may need to see what @rdb thinks about this.

Reply:
Ye, the only reason I changed in a .h file is because the two files that were raising errors had references to that file. Changing that sounds easy. But do you want to wait for @rdb opinion before I change it?

Reply:
I'm fine with putting this in the .cxx file if this is necessary.  However, if MinGW does not define this, are we sure that its getaddrinfo actually supports this flag?

As CFS pointed out in a different comment, don't use `/**` comments here, which are only for docstrings of public interfaces.

Reply:
> I'm fine with putting this in the .cxx file if this is necessary. However, if MinGW does not define this, are we sure that its getaddrinfo actually supports this flag?

It's not getting defined because of `ws2tcpip.h` has this line `_WIN32_WINNT >= 0x0600` which doesn't trigger in my computer. The tests work fine adding the lines above so I assume mingw works with fine if `AI_ADDRCONFIG` and `AI_V4MAPPED` gets defined.

Reply:
Ah, okay.  We will be setting `_WIN32_WINNT` to `0x0600` in Panda in about a month, so I don't really mind what hack we use in the meantime.